privacy_evaluator/metrics/accuracy.py

```diff
@@ -0,0 +1,34 @@
+def accuracy_difference(train_accuracy, test_accuracy):
```

Comment:
I would rename here to train_test_accuracy_difference (such that the name of the function already indicates that it is train-test)

Reply:
This is actually code from team 1 and they fixed it as far as we know.